doc/user/content/release-notes.md

```diff
@@ -19,6 +19,69 @@ the PR description instead. They will be migrated here during the release
 process by the release notes team.
 {{< /comment >}}
 
+{{% version-header v0.22.0 %}}
+
+- **Breaking change.** Standardize handling of the following [nonpure functions](/sql/functions/#pure-and-nonpure-functions) {{% gh 10445 %}}:
+
+  - `current_database`
+  - `current_timestamp`
+  - `current_role`
+  - `current_schema`
+  - `current_schemas`
+  - `current_user`
+  - `mz_cluster_id`
+  - `mz_logical_timestamp`
+  - `mz_uptime`
+  - `mz_version`
+  - `session_user`
+  - `pg_backend_pid`
+  - `pg_postmaster_start_time`
+  - `version`
+
+  Materialize now allows use of nonpure functions in views, but will refuse to
+  create an index that directly or indirectly depends on a nonpure function.
+  The one exception is [`mz_logical_timestamp`], which can be used in limited
+  contexts in a materialized view as a [temporal filter](/guides/temporal-filters).
+
+  Previously `current_timestamp`, `mz_logical_timestamp`, and `mz_uptime` were
+  incorrectly disallowed in unmaterialized views, while the remaining nonpure
+  functions were incorrectly allowed in materialized views.
+
+- **Breaking change.** Store days separately in [`interval`].
+Unlike the previous version, Hours are never converted to days. This
+means that: an Interval of 24 hours will not be equal to an Interval
+of 1 day, you cannot subtract hours from days, and when ordering
+Intervals `d days > h hours` for all `d,h` {{% gh 10708 %}}.
+
+- **Breaking change.** Print all negative [`interval`]
+units as plural (e.g., `-1 days` will be printed instead of `-1 day`). This
+matches the behavior of PostgreSQL.
+
+- **Breaking change.** Round microsecond field of
+[`interval`] to 6 places before applying the
+given precision. For example `INTERVAL '1.2345649' SECOND(5)` will be
+rounded to `00:00:01.23457`, not `00:00:01.23456`. This matches the
+behavior of PostgreSQL.
+
+- Add the following aliases to Interval parsing: `yr`, `yrs`, `hr`, `hrs`, `min`, `mins`, `sec`, `secs`.
+
+- Add the [`justify_days`](/sql/functions/justify-days), [`justify_hours`](/sql/functions/justify-hours), and [`justify_interval`](/sql/functions/justify-interval) functions.
+
+- Change the range of the `oid` type from [-231, 231 - 1] to [0, 232 - 1] to match PostgreSQL.
+
+- Fix a panic when materializing a source that has been renamed since it was last materialized {{% gh 10904 %}}.
+
+- Change the claimed PostgreSQL version returned by the `version()` function to 9.5 to match the values of the `server_version` and `server_version_num session` parameters.
+
+- Fix data loss bug of in [Postgres sources] introduced in version v0.20.0 {{% gh 10981 %}}.
+
+- More precisely infer the output type of numeric division and modulo operations as `NOT NULL` when the input arguments are `NOT NULL`.
```

Comment:
This is a little in the weeds. I'll leave to you to though, @morsapaes, to decide whether to toss.

Reply:
Yeah, I was on the fence about this one but will drop it.